make.bat

```diff
@@ -28,27 +27,25 @@ if not exist "%gccpath%" (
     goto:msvcstrap
 )
 
-gcc -std=c99 -municode -w -o v2.exe vc\v_win.c
+gcc -std=c99 -municode -w -o v.exe vc\v_win.c
 if %ERRORLEVEL% NEQ 0 (
     echo gcc failed to compile - Create an issue at 'https://github.com/vlang'
```

Comment:
I would also unify error output. We can goto `:compileerror` here instead of exiting with 1.

Reply:
Because always run make.bat compilation error, only to modify it, I hope there is a better way.